site/installing.md

```diff
@@ -172,22 +178,22 @@ To enable DaemonSets in an existing cluster, add the `--runtime-config=extension
 
 **Install Scope on Your Cluster**
 
-It is recommended that you run Scope natively in your Kubernetes cluster using `kubectl` with launch URL as show below.
+You can run Scope natively in your Kubernetes cluster using `kubectl` with launch URL as show below.
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-    kubectl create -f 'https://scope.weave.works/launch/k8s/weavescope.yaml' --validate=false
-
-> The `--validate=false` flag is currently required due to a bug in Kubernetes (see
-[kubernetes/kubernetes#24089](https://github.com/kubernetes/kubernetes/issues/24089) for more details
+    kubectl create -f 'https://cloud.weave.works/launch/k8s/weavescope.yaml' --validate=false
 
 Allowable parameters for the launcher URL:
 
-- `v` - Weave Scope version or tag, e.g. `latest` or `0.15.0`, current release is the default
-- `service-token` - Weave Scope Cloud Service token
+- `v` - Weave Scope version or tag, e.g. `latest` current release is the default
+- `service-token` - Weave Cloud Service token (necessary if you want to run Weave Cloud)
```

Comment:
Can you maybe elaborate on how to find the token? Also, it may be helpful to include a ` kubectl create` example using the with the `service-token` parameter